doc/examples.xml

```diff
@@ -533,6 +533,43 @@ true]]></Example>
 </ManSection>
 <#/GAPDoc>
 
+<#GAPDoc Label="BurntPancakeGraph">
+<ManSection>
+  <Oper Name="BurntPancakeGraph" Arg="[filt, ]n"/>
+  <Returns>A digraph.</Returns>
+  <Description>
+    If <A>n</A> is a positive integer, then this operation returns
+    the <E>Burnt Pancake graph</E> with <M>n!</M> vertices and <M>n!(n - 1)</M>
+    directed edges. The <M>n</M>th Pancake graph is the Cayley graph of the
+    hyperoctahedral group acting on <C>[<A>-n</A> .. -1, 1 .. <A>n</A>]</C> with respect to the
+    generating set consisting of the <Q>prefix reversals</Q>, which are defined in exactly
+    the same way as in  <Ref Attr="PancakeGraph"/>. The hyperoctahedral group consists of
+    permutations <M>p</M> acting on <C>[<A>-n</A> .. -1, 1 .. <A>n</A>]</C>, where the image
+    of any arbitrary point <M>i</M> in <C>[<A>-n</A> .. -1, 1 .. <A>n</A>]</C> is equal to the
+    negative of the image of <M>-i</M> under <M>p</M>.
```

Comment:
It'd be good to mention that in GAP permutations act on the positive integers only, and as such the permutations of -n to n (without 0) are represented by ???

Reply:
I don't know how I didn't push this, sorry, I wrote it and then deleted it by mistake probably. Fixing this now :)